resources/js/pages/Home.vue

```diff
@@ -15,46 +20,92 @@
     import Expense from '../components/Expense.vue';
     import Nav from '../components/Nav.vue';
     import Summary from '../components/Summary.vue';
+    import { ADD_ENTRY, DELETE_ENTRY, RESET_TOTAL, UPDATE_ENTRY } from '../constants';
+    import { DateTime } from 'luxon';
 
     export default {
       components: {
         Expense,
         Summary,
         Nav,
       },
+
+      created() {
```

Comment:
It looks like this component is never used?

Reply:
I used this while scaffolding but I removed it just now